src/openrct2-ui/windows/Ride.cpp

```diff
@@ -3241,7 +3241,7 @@ static void window_ride_mode_tweak_increase(rct_window* w)
         maxValue = 255;
     }
 
-    uint8_t increment = ride->mode == RIDE_MODE_DODGEMS ? 10 : 1;
+    uint8_t increment = ride->mode == static_cast<uint8_t>(RideMode::RIDE_MODE_DODGEMS) ? 10 : 1;
```

Comment:
Change the type of ride->mode to the new type

Reply:
Ok, and for the `uint64_t RideModes;` in 'struct RideTypeDescriptor' should I change it to 'RideMode[8]' or 'std::vector<RideMode>' or leave it as is?

Reply:
Hmm. It's an array of available modes using each mode as a flag. Best to keep the type uint64_t but there is going to be a ton of static casts to make that work. @gymnasiast might have a better idea.